cmd/thanos/rule.go

```diff
@@ -511,13 +514,14 @@ func runRule(
 			cancel()
 		})
 	}
+
 	// Start gRPC server.
 	{
 		l, err := net.Listen("tcp", grpcBindAddr)
 		if err != nil {
 			return errors.Wrap(err, "listen API address")
 		}
-		logger := log.With(logger, "component", component.Rule.String())
+		logger := log.With(logger, "component", "store")
```

Comment:
is this change intended?

Reply:
no and have no idea how that happened. thanks I'll revert it